include/inviwo/core/datastructures/histogram.h

```diff
@@ -65,106 +57,13 @@ struct IVW_CORE_API Histogram1D {
     size_t totalCounts{0};
     size_t maxCount{0};
     DataMapper dataMap{};
+    dvec2 effectiveDataRange{0.0};
```

Comment:
Those are needed in case one wants to compute the bin centers in either DataRange or ValueRange as done in the HistogramToDataFrame processors.
https://github.com/inviwo/inviwo/blob/0a4d88da3abc8c091dd74a389973fdd2612e27c5/modules/volume/src/processors/histogramtodataframe.cpp#L76-L87

Reply:
But why it this not needed in the TF editor than? you did not change that right? 

Think you need to explain how this is related to just dividing the datarange in nbins steps.

Reply:
Adjusted the TF editor accordingly.